keras_cv/models/backbones/csp_darknet/csp_darknet_backbone_test.py

```diff
@@ -131,7 +131,7 @@ def test_create_backbone_model_with_level_config(self):
         ("S", csp_darknet_backbone.CSPDarkNetSBackbone),
         ("M", csp_darknet_backbone.CSPDarkNetMBackbone),
         ("L", csp_darknet_backbone.CSPDarkNetLBackbone),
-        ("X", csp_darknet_backbone.CSPDarkNetXBackbone),
+        ("X", csp_darknet_backbone.CSPDarkNetXLBackbone),
```

Comment:
Probably also worth making this string "XL", although it doesn't really matter a lot

Reply:
Done.